client/keys/show_test.go

```diff
@@ -24,7 +25,7 @@ func Test_multiSigKey_Properties(t *testing.T) {
 		1,
 		[]cryptotypes.PubKey{tmpKey1.PubKey()},
 	)
-	tmp, err := keyring.NewMultiInfo("myMultisig", pk)
+	tmp, err := keyring.NewLegacyMultiInfo("myMultisig", pk)
```

Comment:
Why are we creating a legacy info here? Does this test run migration?

Reply:
Done